catalog/resource_sql_table.go

```diff
@@ -488,10 +488,23 @@ func columnChangesCustomizeDiff(d *schema.ResourceDiff, newTable *SqlTableInfo)
 	return nil
 }
 
+var columnTypeAliases = map[string]string{
```

Comment:
there are additional aliases that are less used
- float | real - https://docs.databricks.com/en/sql/language-manual/data-types/float-type.html
- decimal | dec | numeric - https://docs.databricks.com/en/sql/language-manual/data-types/decimal-type.html
- smallint | short - https://docs.databricks.com/en/sql/language-manual/data-types/smallint-type.html
- tinyint | byte - https://docs.databricks.com/en/sql/language-manual/data-types/tinyint-type.html

Reply:
`decimal` technically must be specified with precision & scale, i.e. `decimal(p,s)`. `decimal` just defaults to `decimal(10,0)`